l1-contracts/contracts/bridge/ntv/NativeTokenVault.sol

```diff
@@ -111,7 +110,7 @@ abstract contract NativeTokenVault is INativeTokenVault, IAssetHandler, Ownable2
     ) external payable override onlyAssetRouter whenNotPaused {
         address receiver;
         uint256 amount;
-        if (isTokenNative[_assetId]) {
+        if (originChainId[_assetId] == block.chainid) {
```

Comment:
please add a comment that explains why is it safe even if the token has been bridged prior to this update (I suspect, it is an indirect check when we deploy the token)